packages/lib-react-components/src/Modal/Modal.js

```diff
@@ -6,6 +6,10 @@ import ModalBody from './components/ModalBody'
 import ModalHeading from './components/ModalHeading'
 
 const Modal = forwardRef(function ({
+  bodyBackground = {
+    dark: 'dark-5',
+    light: 'neutral-6'
+  },
```

Comment:
Suggestion to define this style object outside of the Modal component like:
```
const defaultBg = {
    dark: 'dark-5',
    light: 'neutral-6'
}
```

You could pass it to the parent Box where `background` is defined in this file too. Similar strategy to:
https://github.com/zooniverse/front-end-monorepo/blob/cf0fcfc2630707555294760c7af07fd58e76768c/packages/lib-react-components/src/AuthModal/components/LoginForm/LoginForm.js#L7-L10